aten/src/ATen/test/vec_test_all_types.cpp

```diff
@@ -978,26 +978,6 @@ namespace {
             b[i] = b[i - 1] + (T)(1.0);
         }
     }
-    template<>
```

Comment:
Is this just testing the template instationation? Or was this code snippit suppose to be used somewhere in a test and wasn't due to a bug?